### PR TITLE
Initial layout

### PR DESCRIPTION
**Pull request by limavfabio**
This PR adds the initial implementation of the React Capstone Project, Fruity Vice. It's a web app for viewing information about diverse fruits. Built using React, React Router and Redux.

For purposes of documentation, here is a short video explaining some features of the app: https://www.loom.com/share/4e983366108f4796b35a2da220136f34

Here is the link to the deployed app:
https://curious-begonia-4dc710.netlify.app/

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,46 +1,55 @@
-# Getting Started with Create React App and Redux
+# Fruity App
```
Comment (Whoistolu) [inline README.md:1]: - [x] [REQUIRED] Great job so far @limavfabio 👍🏽 . However, I would recommend that you 

1. Record a video presenting the features of the project
2. The maximum length of the video is 5 minutes
3. The video contains a clear and objective description of the project. The topic and purpose of the project are made clear.
4. The video contains a project demo, highlighting all the project features.
5. The video contains a highlight of a piece of code that you are proud of making.

- [x] [REQUIRED] Kindly add an "Acknowledgements" section that gives appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the author of the [original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/).




These recommendations are part of the project requirement and will make your project more professional.

File: README.md [hunk 2]
```diff
@@ -1,46 +1,55 @@
...
+- GitHub: (https://github.com/limavfabio)
 
-You don't have to ever use `eject`. The curated feature set is suitable for small and middle deployments, and you shouldn't feel obligated to use this feature. However we understand that this tool wouldn't be useful if you couldn't customize it when you are ready for it.
+# Contributing
 
-## Learn More
+- Please check the issues page
 
-You can learn more in the [Create React App documentation](https://facebook.github.io/create-react-app/docs/getting-started).
+# Live Demo
```
Comment (Whoistolu) [inline README.md:49]: - [x] [REQUIRED] It is also part of the project requirements that you deploy the application using Heroku, Netlify, or GitHub Pages. kindly ensure that the project is deployed correctly, the home and detail page works as expected. Then attach a link to the deployed project to this part of the README.

File: src/app/store.js [hunk 3]
```diff
@@ -1,8 +1,10 @@
...
-import counterReducer from '../features/counter/counterSlice';
+import { fruitsApiSlice } from '../features/fruits/fruitsApiSlice';
 
 export const store = configureStore({
   reducer: {
-    counter: counterReducer,
+    [fruitsApiSlice.reducerPath]: fruitsApiSlice.reducer,
   },
+  middleware: (getDefaultMiddleware) =>
+    getDefaultMiddleware().concat(fruitsApiSlice.middleware),
```
Comment (Whoistolu) [inline src/app/store.js:9]: - [x] [REQUIRED] Kindly ensure that this project uses Redux for state management and data retrieved from the API are stored in the Redux store correctly. This is a hard requirement.

File: src/routes/FruitDetails.jsx [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+            <div>
+              <strong>Fat:</strong> {fruit.nutritions.fat}
+            </div>
+            <div>
+              <strong>Protein:</strong> {fruit.nutritions.protein}
+            </div>
+            <div>
+              <strong>Sugar:</strong> {fruit.nutritions.sugar}
+            </div>
+          </div>
```
Comment (Whoistolu) [inline src/routes/FruitDetails.jsx:53]: - [x] [REQUIRED] Kindly note that the project requirement states that you should select one main color for the app and be consistent. Kindly remove this color `#bde0fe` and replace it with a shade of the major color you chose just like you did on the `Home` page.

Figma Design   |  Your Implementation 
:-------------------------:|:-------------------------:
Screenshot from 2022-10-06 23-15-10 | Screenshot from 2022-10-06 23-14-51

File: src/components/Header.jsx [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+import style from '../style/Header.module.css';
+import { useNavigate } from 'react-router-dom';
+
+export default function Header() {
+  const navigate = useNavigate();
+  return (
+    <div className={style.container}>
+      <button onClick={() => navigate(-1)} className={style.back}>
+        Back
+      </button>
```
Comment (Whoistolu) [inline src/components/Header.jsx]: - [x] [REQUIRED] Kindly ensure that the `Back` button is not been displayed on the `Home` page for a better UI.

**Review by Whoistolu — CHANGES_REQUESTED** [review]
Hi @limavfabio 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️

- [x]  _Check the inline comments under the review for sections to improve._
- #### I will leave these comments here because they are not in your files changes.
- [x] [REQUIRED] Kindly add a Filter stateless component to filter data retrieved from the API as stated by the project requirement.
- [x] [REQUIRED] Kindly ensure that this App has unit tests for several components of the app and some pure functions in Redux code and also the App has integration tests using React Testing Library.

### Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

### N/A

Cheers and Happy coding!

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @whoistolu in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 6]
```diff
@@ -1,46 +1,59 @@
...
```
Comment (codepantha) [inline README.md:49]: - [x] As requested by the previous reviewer, please ensure to deploy your application using Heroku, Netlify, or GitHub Pages. kindly ensure that the project is deployed correctly, the home and detail page works as expected. Then attach a link to the deployed project to this part of the README

**Review by codepantha — CHANGES_REQUESTED** [review]
# REQUIRES CHANGES ♻️ 

## Hi @limavfabio 👋, 

Great work on making the changes requested by the previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet.

## Highlights 🎯 
✔️ Descriptive PR
✔️ Used correct GitFlow
✔️ No linter errors

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by youmari — APPROVED** [review]
Hi @limavfabio , 

In my opinion, your project is ready for the presentation!  There is nothing else to say other than... it's time to merge it :shipit: 


Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**